site_scons/prereq_tools/base.py

```diff
@@ -705,7 +705,7 @@ def __init__(self, env, variables, config_file=None, arch=None):
             self.configs.read(config_file)
 
         self.installed = env.subst("$USE_INSTALLED").split(",")
-        self.exclude = env.subst("$EXCLUDE").split(",")
+        self.include = env.subst("$INCLUDE").split(" ")
```

Comment:
This used to be " " but is not ",", is that expected?
```suggestion
        self.include = env.subst("$INCLUDE").split(",")
```

Reply:
Yes, this is expected. Although, I am not sure why but if you use two or more values:

`$ scons INCLUDE=psm2,psm3`

The env var `INCLUDE` will have `psm2 psm3`. So, I split the values by space.

Reply:
Probably the old code never worked because we only had one option